tsconfig.json

```diff
@@ -26,6 +26,7 @@
             "@components": ["./src/components/index"],
             "@pages": ["./src/pages/index"],
             "@services": ["./src/services/index"],
+            "@providers": ["./src/providers/index"],
             "@utils": ["./src/utils/index"],
```

Comment:
```suggestion
            "@utils": ["./src/utils/index"]
```
json does not have ending "," for last entry

Reply:
json "does not" need to have ending "," for last entry?

Reply:
if so, i gonna change for every other json in the folder as well